test/libsolidity/semanticTests/types/mapping_enum_key_library_encoderV2.sol

```diff
@@ -0,0 +1,39 @@
+pragma experimental ABIEncoderV2;
+
+enum E { A, B, C }
+library L {
+    function get(mapping(E => uint8) storage table, E k) external returns (uint8) {
+        return table[k];
+    }
+    function set(mapping(E => uint8) storage table, E k, uint8 v) external {
+        table[k] = v;
+    }
+}
+contract test {
+    mapping(E => uint8) table;
+    function get(E k) public returns (uint8 v) {
+        return L.get(table, k);
+    }
+    function set(E k, uint8 v) public {
+        L.set(table, k, v);
+    }
+}
+// ====
+// EVMVersion: >=byzantium
+// ----
+// library: L
+// get(uint8): 0 -> 0
+// get(uint8): 0x01 -> 0
+// get(uint8): 0xa7 -> FAILURE
```

Comment:
Why no code?

Reply:
It's the abi decoder that creates the error here, which is an "old-style" revert without message (unless you activated compiler-generated messages)